python/django/django-realworld/django-realworld-example-app/requirements.txt

```diff
@@ -0,0 +1,7 @@
+Django==1.10.5
+django-cors-middleware==1.3.1
+django-extensions==1.7.1
+djangorestframework==3.4.4
+PyJWT==1.4.2
+six==1.10.0
+git+https://github.com/DataDog/dd-trace-py.git@598afbca483a59c7178ac010836d82dd38869140
```

Comment:
We should wait until we merge and release latest `ddtrace` and use that instead of this here